package/utils/omnia-eeprom/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2024 Marek Behún <kabel@kernel.org>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=omnia-eeprom
+PKG_VERSION:=v0.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/$(PKG_NAME)/-/archive/$(PKG_VERSION)/
+PKG_HASH:=6f949d0b8080adca8bae088774ce615b563ba6ec2807cce97ee6769b4eee7bbf
+
+PKG_MAINTAINER:=Marek Behun <kabel@kernel.org>
+PKG_LICENSE:=GPL-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/omnia-eeprom
+  SECTION:=utils
+  CATEGORY:=Utilities
+  URL:=https://gitlab.nic.cz/turris/$(PKG_NAME)
+  TITLE:=CZ.NIC Turris Omnia EEPROM accessing utility
+  DEPENDS:=@TARGET_mvebu_cortexa9_DEVICE_cznic_turris-omnia
+endef
+
+define Package/omnia-eeprom/description
+This package contains the omnia-eeprom utility, which allows you to display
+and update EEPROM fields on the Turris Omnia router.
+The EEPROM is normally not meant to be updated by users, but there are some
+exceptions where it might be useful.
```

Comment:
I am thinking loud. Do we really need to have this package preinstalled for all Turris Omnia routers? I know they are huge in every spec than usual OpenWrt router, but for now, the only benefit of this package is that it helps your customers who have issues with recent U-boot loader on their older router revisions. But if the user is updating U-boot, he is skilled enough and knows what he does, thus shouldnt this be in the packages feed?

Reply:
The issue this package is meant to solve is *random crash due to DDR errors*. If a router ends up in this state, we want to have the package available. Having to download the package and install it takes some CPU computations and network controller accesses to DDR, which just makes it more probable that a DDR error will occur and the router will crash before the user is able to execute `omnia-eeprom`.

Combined with the fact that this package compiles to a 12.5 KiB binary, I think it can safely be installed for all Omnias.